build/yaml/functionalTestsBotsBuild.yml

```diff
@@ -0,0 +1,233 @@
+steps:
+# Note: Template ci-build-steps.yml is not supported in macOS because it calls VSBuild@1 in order to build the Windows-only ASP.NET Desktop assemblies.
```

Comment:
This comment don't seem to apply, remove?

Reply:
removed the comment